contrib/pom.xml

```diff
@@ -0,0 +1,360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.google.adk</groupId>
+    <artifactId>google-adk-contrib</artifactId>
+    <version>0.1.0</version><!-- {x-version-update:google-adk:current} -->
+    <packaging>jar</packaging>
+    <name>Agent Development Kit - Contributions</name>
+    <url>https://github.com/google/adk-java</url>
+    <licenses>
```

Comment:
Seeing this here makes me think that we should have a root parent POM to define everything that's not specific to core or dev or contrib. I can propose a PR for that, if there is agreement that this would make sense?

Reply:
I'm not an expert in Maven, but I definitely agree we need to make the build a little more modular to avoid repeating things, and more clearly separating the different artifacts.